dockstore-client/src/main/java/io/dockstore/client/cli/nested/DepCommand.java

```diff
@@ -46,7 +53,7 @@ public static boolean handleDepCommand(String[] args) {
         private String clientVersion = getClass().getPackage().getImplementationVersion();
         @Parameter(names = "--python-version", description = "Python version")
         private String pythonVersion = "2";
-        @Parameter(names = "--runner", description = "tool/workflow runner")
+        @Parameter(names = "--runner", description = "tool/workflow runner. Available options: 'cwltool'")
```

Comment:
Seems like more of an "option" than "options" ;)
Just remove the option and default to cwltool for the user facing CLI 
(we can leave the option there on the web service for future expansion)